styles/_footer.scss

```diff
@@ -1,20 +1,195 @@
 /***** Footer *****/
-.footer {
-  border-top: 1px solid $border-color;
-  margin-top: 60px;
-  padding: 30px 0;
+footer {
+  margin: 0 auto;
+  @include fluid-value( padding-top, $min-width, $max-width, $whitespace-medium, $whitespace-medium--desktop );
+  max-width: $max-width-page-container;
+  background-color: $primary--light;
+  color: $white;
 
-  a { color: $secondary-text-color; }
+  .footer-inner {
```

Comment:
Is there a reason you didn't use regular `BEM` here?

Reply:
I was adapting their code. In retrospect, if I'm going to touch a file I really should convert it to our style.